src/main/kotlin/no/nav/syfo/persistering/api/SendRegisterPapirSykmeldingOppgaveRest.kt

```diff
@@ -201,31 +197,12 @@ fun Route.sendPapirSykmeldingManuellOppgave(
                                     }
                                 }
                                 Status.MANUAL_PROCESSING -> {
-                                    if (manuellOppgaveService.ferdigstillSmRegistering(oppgaveId) > 0) {
-                                        handleManuellOppgave(
-                                            receivedSykmelding = receivedSykmelding,
-                                            kafkaRecievedSykmeldingProducer = kafkaRecievedSykmeldingProducer,
-                                            loggingMeta = loggingMeta,
-                                            session = session,
-                                            syfoserviceProducer = syfoserviceProducer,
-                                            oppgaveClient = oppgaveClient,
-                                            dokArkivClient = dokArkivClient,
-                                            sykmeldingId = sykmeldingId,
-                                            journalpostId = journalpostId,
-                                            healthInformation = healthInformation,
-                                            oppgaveId = oppgaveId,
-                                            validationResult = validationResult,
-                                            kafkaManuelTaskProducer = kafkaManuelTaskProducer,
-                                            kafkaValidationResultProducer = kafkaValidationResultProducer
-                                        )
-                                        call.respond(HttpStatusCode.NoContent)
-                                    } else {
-                                        log.error(
-                                            "Ferdigstilling av papirsykmeldinger manuell registering i db feilet {}",
-                                            StructuredArguments.keyValue("oppgaveId", oppgaveId)
-                                        )
-                                        call.respond(HttpStatusCode.InternalServerError)
-                                    }
+                                    log.info(
```

Comment:
Dette her er for at vi ikke skal slippe gjennom sykmeldinger som ellers ville gått til manuell behandling siden vi antar at de som jobber i smregistrering også gjør den manuelle behandlingen? :)

Vi bør i så fall gjøre noe med feilhåndteringen her sånn at frontend får bedre beskjed om hva som er galt, men det tenker jeg er en egen pr :D

Reply:
De får feilmeldingene fra regel-tjenesten :)